app/presenters/teachers/lesson_contents_presenter.rb

```diff
@@ -22,6 +22,7 @@ def initialize(lesson, teacher)
       @contents = lesson
         .lesson_parts
         .each_with_object({}) { |lesson_part, hash| hash[lesson_part] = lesson_part.activity_for(teacher) }
+        .reject { |_, activity| activity.nil? }
```

Comment:
I know not all lesson_parts have an activity at the minute but do you think we should fail if there's a lesson_part without an activity when we go live?
Maybe make this reject conditional on an environment variable `LESSON_CONTENT_WIP` or something? 🤷‍♂️ Or add a note why this is here?

Reply:
I was wondering that. I think that we should probably have a state machine governing every CCP (or maybe unit) and before publishing it it should make these checks to ensure every lesson part has at least one activity, every unit has a lesson etc.